packages/i18nify-js/src/modules/geo/getFlagOfCountry.ts

```diff
@@ -1,7 +1,7 @@
 import { CountryCodeType, GetFlagReturnType } from '../types';
 import { withErrorBoundary } from '../../common/errorBoundary';
 import { FLAG_4X3_BASE_PATH, FLAG_BASE_PATH } from './constants';
-import { isCountryValid } from './utils';
+import { isCountryValidForFlags } from './utils';
```

Comment:
Why do we have to change the name of this utility here ?

Reply:
This was done for earlier approach where we were maintaining different country list for flags. Will revert this.